kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml

```diff
@@ -508,6 +508,32 @@ paths:
                 items:
                   $ref: '#/components/schemas/ConsumerGroup'
 
+  /api/clusters/{clusterName}/consumer-groups/{id}/reset-offsets:
```

Comment:
maybe we could do it like 
```/api/clusters/{clusterName}/consumer-groups/{id}/offsets:```

Reply:
ok to rename, but this operation called "reset-offsets" in kafka-consumer-groups console utility